Problem with apache build

there is a problem with the apache build target.
there seems to be a hardcoded vhost name ? luxembourg-geomapfish is not a variable I have set somewhere
```
echo "Include /var/www/vhosts/geoportailv3/apache/*.conf" > /var/www/vhosts/luxembourg-geomapfish/conf/main.conf
/bin/sh: /var/www/vhosts/luxembourg-geomapfish/conf/main.conf: No such file or directory
make: *** [/var/www/vhosts/luxembourg-geomapfish/conf/main.conf] Error 1
```

--- a/CONST_Makefile
+++ b/CONST_Makefile
@@ -232,6 +232,7 @@ PRINT_REQUIREMENT += $(PRINT_BASE_DIR)/$(PRINT_BASE_WAR)
 APACHE_ENTRY_POINT ?= /$(INSTANCE_ID)/
 APACHE_VHOST ?= $(PACKAGE)
 APACHE_CONF_DIR ?= /var/www/vhosts/$(APACHE_VHOST)/conf
+APACHE_GRACEFUL ?= sudo /usr/sbin/apache2ctl graceful
 CONF_FILES_IN = $(shell ls -1 apache/*.conf.in 2> /dev/null)
 CONF_FILES_MAKO = $(shell ls -1 apache/*.conf.mako 2> /dev/null)
 CONF_FILES_JINJA = $(shell ls -1 apache/*.conf.jinja 2> /dev/null)
@@ -702,5 +703,5 @@ $(APACHE_CONF_DIR)/$(INSTANCE_ID).conf:
 	echo "Include $(shell pwd)/apache/*.conf" > $@
 
 .build/apache.timestamp: .build/config-$(INSTANCE_ID).timestamp $(CONF_FILES) $(PY_FILES) $(APACHE_CONF_DIR)/$(INSTANCE_ID).conf .build/requirements.timestamp
-	sudo /usr/sbin/apache2ctl graceful
+	$(APACHE_GRACEFUL)
 	touch $@

--- a/CONST_Makefile
+++ b/CONST_Makefile
@@ -213,6 +213,8 @@ PRINT_REQUIREMENT += \
 	$(PRINT_BASE_DIR)/WEB-INF/classes/logback.xml \
 	$(PRINT_BASE_DIR)/WEB-INF/classes/mapfish-spring-application-context-override.xml
 TOMCAT_SERVICE_COMMAND ?= sudo /etc/init.d/tomcat-tomcat1
+TOMCAT_STOP_COMMAND ?= $(TOMCAT_SERVICE_COMMAND) stop
+TOMCAT_START_COMMAND ?= $(TOMCAT_SERVICE_COMMAND) start
 
 PRINT_REQUIREMENT += $(shell find $(PRINT_BASE_DIR)/print-apps 2> /dev/null)
 endif
@@ -664,12 +666,12 @@ $(PRINT_OUTPUT)/$(PRINT_WAR): $(PRINT_REQUIREMENT)
 	cp $(PRINT_BASE_DIR)/$(PRINT_BASE_WAR) /tmp/$(PRINT_WAR)
 	cd $(PRINT_BASE_DIR) && jar -uf /tmp/$(PRINT_WAR) $(PRINT_INPUT)
 	chmod g+r,o+r /tmp/$(PRINT_WAR)
-ifneq ($(TOMCAT_SERVICE_COMMAND),)
-	$(TOMCAT_SERVICE_COMMAND) stop
+ifneq ($(TOMCAT_STOP_COMMAND),)
+	$(TOMCAT_STOP_COMMAND)
 endif
 	mv /tmp/$(PRINT_WAR) $(PRINT_OUTPUT)
-ifneq ($(TOMCAT_SERVICE_COMMAND),)
-	$(TOMCAT_SERVICE_COMMAND) start
+ifneq ($(TOMCAT_START_COMMAND),)
+	$(TOMCAT_START_COMMAND)
 endif
 
 print/WEB-INF/lib/jasperreports-functions-5.5.0.jar:

